ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-segment-security.cfg

```diff
@@ -0,0 +1,55 @@
+!RANCID-CONTENT-TYPE: arista
+!
+transceiver qsfp default-mode 4x10G
+!
+hostname router-segment-security
+!
+no enable password
+no aaa root
+!
+interface Management1
+   description oob_management
+   vrf MGMT
+   ip address 10.73.255.122/24
+!
+router segment-security
+   no shutdown
+   !
+   policy POLICY-TEST1
+      10 application APP-TEST-1 action forward
+      20 application APP-TEST-2 action drop stateless log
+      25 application APP-TEST-3 action redirect next-hop 198.51.100.1 stateless
+   !
+   vrf default
```

Comment:
Here, we have implemented natural_sort for vrf name, but in EOS the config is arranging the name in `upper case` first then `small case`

In this case - `SECURE` first then `default`

```
router segment-security
   segment policy
      definition
         match interface Ethernet3
   !
   policy po1
      20 application app2 action drop log
      30 application app3 action drop log
      40 application app1 action drop log
   !
   vrf DEFAULT
      segment seg2
         definition
            match covered prefix-list ipv4 pre
   !
   vrf SECURE
      segment SEGMENT-TEST1
         definition
            match interface Ethernet1
            match covered prefix-list ipv4 PREFIX-LIST10
   !
   vrf default
      segment SEGMENT-TEST1
         definition
            match prefix-ipv4 MATCH-LIST10
         !
         policies
            from MATCH-LIST22 policy POLICY-TEST1
      !
      segment seg1
   !
   vrf hello
      segment seg3
         definition
            match interface Port-Channel10
   !
   vrf vrf10
      segment seg1
         definition
            match interface Ethernet2 Ethernet5
```

Reply:
I don't think we have tested that anywhere else. I suspect EOS would do the same everywhere...

Reply:
it does e.g.
```
vrf instance 1234
!
vrf instance MGMT
!
vrf instance TEST123
!
vrf instance test123
!
vrf instance zzz
!
```

Reply:
Okay, then we can keep as is (natural_sort)